src/Microsoft.Sbom.Api/Utils/ComponentDetectorCachedExecutor.cs

```diff
@@ -5,7 +5,9 @@
 using System.Collections.Concurrent;
 using System.Threading.Tasks;
 using Microsoft.ComponentDetection.Contracts.BcdeModels;
+using Microsoft.ComponentDetection.Orchestrator.Commands;
 using Serilog;
+using Spectre.Console.Cli;
```

Comment:
I assume you're specifying this `using` to access `ScanSettings` object, but it seems like for `IComponentDetector.cs` only `using Microsoft.ComponentDetection.Orchestrator.Commands` is sufficient. Are you sure we have to add and use `Spectre.Console.Cli` explicitly?

Reply:
It won't let me do ScanSettings.ToString() If I don't include this. Not entirely sure why. I tried to get the hash of the ScanSettings object directly but then the cachedExecutor wasn't working properly.